src/main/java/cz/cvut/kbss/termit/service/repository/UserRoleRepositoryService.java

```diff
@@ -37,4 +37,8 @@ public UserRoleRepositoryService(UserRoleDao dao) {
     public List<UserRole> findAll() {
         return dao.findAll();
     }
+
+    public UserRole find(cz.cvut.kbss.termit.security.model.UserRole userRole) {
```

Comment:
Check that the result returned by DAO is not empty (see my comment in DAO). See other services and their `findRequired` methods.